lib/rucio/core/rule.py

```diff
@@ -3261,47 +3274,57 @@ def __create_recipents_list(rse_expression, filter_=None, *, session: "Session")
                 try:
                     email = get_account(account=account, session=session).email
                     if email:
-                        recipents.append((email, account))
+                        recipients.append((email, account))
                 except Exception:
                     pass
 
     # LOCALGROUPDISK/LOCALGROUPTAPE
-    if not recipents:
+    if not recipients:
         for rse in parse_expression(rse_expression, filter_=filter_, session=session):
             rse_attr = list_rse_attributes(rse_id=rse['id'], session=session)
             if rse_attr.get('type', '') in ('LOCALGROUPDISK', 'LOCALGROUPTAPE'):
-                accounts = session.query(models.AccountAttrAssociation.account).filter_by(key='country-%s' % rse_attr.get('country', ''),
-                                                                                          value='admin').all()
+                accounts = session.query(
+                    models.AccountAttrAssociation.account
+                ).filter_by(
+                    key=f'country-{rse_attr.get("country", "")}',
+                    value='admin'
+                ).all()
```

Comment:
Can I tempt you into rewriting this and the other query in the SQLAlchemy 2.0 syntax? They should be trivial. Please do so in a separate commit.